### ISSUE TITLE
[BUG] (dxvk) Not a valid wine prefix.

### ISSUE THREAD

**Issue by fishxz** [bottlesdevs/Bottles#256]
**Describe the bug**
when installing dxvk, dxvk reports `/var/home/***/.local/share/bottles/bottles/wow: Not a valid wine prefix.`

```
2021-05-04 15:14:51 INFO: Installing dxvk for bottle: [wow].
/var/home/***/.local/share/bottles/bottles/wow: Not a valid wine prefix.
2021-05-04 15:14:52 INFO: Bottle: [wow] successfully created!
```

**Installation**
 - Method: flatpak
 - Version 1.10.2

**To Reproduce**
1. Click on '+'
2. Enter Name
3. Select Gaming
4. Click on 'Create'
5. See error

**Expected behavior**
dxvk accepts wine prefix as valid

**Desktop (please complete the following information):**
 - OS: fedora silverblue
 - Version 34

**Comment by fishxz** [comment]
after removing the whole bottles folder i get the following error:

```
Datei umbenannt '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll.old'
'/var/home/***/.local/share/bottles/dxvk/dxvk-1.8.1/x64/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll'
Es konnte keine Anwendung gestartet werden, oder es ist keine Anwendung mit der angegebenen Datei verkn�pft.
ShellExecuteEx fehlgeschlagen: Datei nicht gefunden./d3d9.dll: File not found in wine prefix
Failed to add override for d3d9
```

**Comment by mirkobrombin** [comment]
> after removing the whole bottles folder i get the following error:
> 
> ```
> Es konnte keine Anwendung gestartet werden, oder es ist keine Anwendung mit der angegebenen Datei verknüpft.
> ShellExecuteEx fehlgeschlagen: Datei nicht gefunden.
> 
> Datei umbenannt '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll.old'
> '/var/home/***/.local/share/bottles/dxvk/dxvk-1.8.1/x64/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll'
> Es konnte keine Anwendung gestartet werden, oder es ist keine Anwendung mit der angegebenen Datei verkn�pft.
> ShellExecuteEx fehlgeschlagen: Datei nicht gefunden./d3d9.dll: File not found in wine prefix
> Failed to add override for d3d9
> ```

Why are you deleting the whole bottles folder? Close Bottles, remove `~/.local/share/bottles/bottles` and try again.
If it still not working I need to setup a VM to replicate the issue.

**Comment by fishxz** [comment]
this time i only deleted `~/.local/share/bottles/bottles`
```
Gtk-Message: 16:53:07.949: Failed to load module "canberra-gtk-module"
2021-05-04 16:53:08 INFO: Runners found: [chardonnay-6.4]
2021-05-04 16:53:08 INFO: Dxvk found: [dxvk-1.8.1]
2021-05-04 16:53:08 INFO: Connection status: online …
https://raw.githubusercontent.com/bottlesdevs/components/main//index.json
2021-05-04 16:53:08 INFO: Bottles path doens't exist, creating now.
2021-05-04 16:53:08 INFO: Dxvk found: [dxvk-1.8.1]
2021-05-04 16:53:08 INFO: Runners found: [chardonnay-6.4]
2021-05-04 16:53:08 INFO: Connection status: online …
2021-05-04 16:53:08 INFO: Connection status: online …
2021-05-04 16:53:08 INFO: Connection status: online …
2021-05-04 16:53:08 INFO: Connection status: online …
2021-05-04 16:53:08 INFO: Bottles Started!
2021-05-04 16:53:13 INFO: Creating the wineprefix …
wine: created the configuration directory '/var/home/***/.local/share/bottles/bottles/wow'
0048:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
0048:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
0048:err:ole:apartment_get_local_server_stream Failed: 0x80004002
0050:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
0050:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
0050:err:ole:apartment_get_local_server_stream Failed: 0x80004002
0050:err:ole:start_rpcss Failed to open RpcSs service
0040:err:setupapi:SetupDefaultQueueCallbackW copy error 1812 L"@C:\\windows\\system32\\drivers\\wineusb.sys,-1" -> L"C:\\windows\\inf\\wineusb.inf"
0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
wine: failed to open L"C:\\windows\\syswow64\\rundll32.exe": c0000135
wine: configuration in L"/var/home/***/.local/share/bottles/bottles/wow" has been updated.
2021-05-04 16:53:22 INFO: Generating Bottle configuration file …
2021-05-04 16:53:22 INFO: Applying environment: [Gaming] …
2021-05-04 16:53:28 INFO: Installing dxvk …
2021-05-04 16:53:28 INFO: Installing dxvk for bottle: [wow].
Application could not be started, or no application associated with the specified file.
ShellExecuteEx failed: File not found.

renamed '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll.old'
'/var/home/***/.local/share/bottles/dxvk/dxvk-1.8.1/x64/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll'
Application could not be started, or no application associated with the specified file.
ShellExecuteEx failed: File not found./d3d9.dll: File not found in wine prefix
Failed to add override for d3d9
2021-05-04 16:53:29 INFO: Bottle: [wow] successfully created!
2021-05-04 16:53:31 INFO: Bottles found: wow
```

**Comment by mirkobrombin** [comment]
> this time i only deleted `~/.local/share/bottles/bottles`
> 
> ```
> Gtk-Message: 16:53:07.949: Failed to load module "canberra-gtk-module"
> 2021-05-04 16:53:08 INFO: Runners found: [chardonnay-6.4]
> 2021-05-04 16:53:08 INFO: Dxvk found: [dxvk-1.8.1]
> 2021-05-04 16:53:08 INFO: Connection status: online …
> https://raw.githubusercontent.com/bottlesdevs/components/main//index.json
> 2021-05-04 16:53:08 INFO: Bottles path doens't exist, creating now.
> 2021-05-04 16:53:08 INFO: Dxvk found: [dxvk-1.8.1]
> 2021-05-04 16:53:08 INFO: Runners found: [chardonnay-6.4]
> 2021-05-04 16:53:08 INFO: Connection status: online …
> 2021-05-04 16:53:08 INFO: Connection status: online …
> 2021-05-04 16:53:08 INFO: Connection status: online …
> 2021-05-04 16:53:08 INFO: Connection status: online …
> 2021-05-04 16:53:08 INFO: Bottles Started!
> 2021-05-04 16:53:13 INFO: Creating the wineprefix …
> wine: created the configuration directory '/var/home/***/.local/share/bottles/bottles/wow'
> 0048:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
> 0048:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
> 0048:err:ole:apartment_get_local_server_stream Failed: 0x80004002
> 0050:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
> 0050:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
> 0050:err:ole:apartment_get_local_server_stream Failed: 0x80004002
> 0050:err:ole:start_rpcss Failed to open RpcSs service
> 0040:err:setupapi:SetupDefaultQueueCallbackW copy error 1812 L"@C:\\windows\\system32\\drivers\\wineusb.sys,-1" -> L"C:\\windows\\inf\\wineusb.inf"
> 0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 0064:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
> 006c:err:mscoree:LoadLibraryShim error reading registry key for installroot
> wine: failed to open L"C:\\windows\\syswow64\\rundll32.exe": c0000135
> wine: configuration in L"/var/home/***/.local/share/bottles/bottles/wow" has been updated.
> 2021-05-04 16:53:22 INFO: Generating Bottle configuration file …
> 2021-05-04 16:53:22 INFO: Applying environment: [Gaming] …
> 2021-05-04 16:53:28 INFO: Installing dxvk …
> 2021-05-04 16:53:28 INFO: Installing dxvk for bottle: [wow].
> Application could not be started, or no application associated with the specified file.
> ShellExecuteEx failed: File not found.
> 
> renamed '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll.old'
> '/var/home/***/.local/share/bottles/dxvk/dxvk-1.8.1/x64/d3d9.dll' -> '/var/home/***/.local/share/bottles/bottles/wow/dosdevices/c:/windows/system32/d3d9.dll'
> Application could not be started, or no application associated with the specified file.
> ShellExecuteEx failed: File not found./d3d9.dll: File not found in wine prefix
> Failed to add override for d3d9
> 2021-05-04 16:53:29 INFO: Bottle: [wow] successfully created!
> 2021-05-04 16:53:31 INFO: Bottles found: wow
> ```

There is a bug in chardonnay build 6.4. Can you install a runner other than your preferences (e.g. Lutris) and try again? Please note that if you install Proton runner you will not need to install dxvk.

**Comment by mirkobrombin** [comment]
Should be fixed with Chardonnay 6.8.
